SearchInRotatedSortedArray.java

```diff
@@ -0,0 +1,92 @@
+//Time Complexity : O(log(n)) Not sure
+//Space Complexity : O(1)
+// Yes, It's run successfully
```

Comment:
Please update the solution with TC: `O(log(n))`. Also please describe your approach briefly.